Cargo.toml

```diff
@@ -20,6 +20,7 @@ members = [
   "runtimes/*",
   "support",
 ]
+resolver = "2"
```

Comment:
[This shouldn't be necessary](https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html) since it's implied by `edition = "2021"`

Reply:
Good point, will remove that. Thanks for the heads up